### ISSUE TITLE
When anonymous user publishes channel show anonxxxxx username

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/zbay#472]
Should this user be listed as "unknown"?
 
<img width="866" alt="image" src="https://user-images.githubusercontent.com/213678/92664329-aa736f80-f2d1-11ea-80ed-edc9f7bd74ff.png">

**Comment by jkdev601** [comment]
Yes, unregistered user can publish a channel, when he will register a username, information message content will update from Anonymous to registered nickname.

**Comment by holmesworcester** [comment]
Solution: show their "anon2352342" username in this message, and anywhere else where we'd say "Anonymous".
